packages/server/src/lib/ab.ts

```diff
@@ -56,3 +62,51 @@ export const selectVariant = <V extends Variant, T extends Test<V>>(test: T, mvt
 
     return selectWithSeed(mvtId, seed, test.variants);
 };
+
+// Amounts test code should match the equivalent code in support-frontend as closely as possible
+const amountsRandomNumber = (mvtId: number, seed: number): number => {
+    const rng = seedrandom(`${mvtId + seed}`);
```

Comment:
I've just noticed that this is subtly different to the use of `seedrandom` above as it adds together 2 numbers then stringifies! For message tests we concat the test name with the mvtId.
I think having a `seed` value in the test data is unfortunate as we can rely on the test name to not change, but I guess we need to keep this consistent with support-frontend now.

Reply:
Refactored code to make this difference clearer